Makefile

```diff
@@ -175,6 +175,12 @@ deploy: update-mgr-env manifests kustomize ## Deploy controller to the K8s clust
 	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+helm-chart: update-mgr-env manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG} && $(KUSTOMIZE) edit set nameprefix ${MANAGER_NAME_PREFIX}
+	cd config/default && $(KUSTOMIZE) edit set image rbac-proxy=$(RBAC_PROXY_IMG)
+	sed -i 's/^version.*/version: ${VERSION}/g' mychart/Chart.yaml
+	$(KUSTOMIZE) build config/default > mychart/templates/template.yaml
```

Comment:
should rename mychart to charts? it's a common name mostly used

Reply:
renamed it to charts